frontend/src/modules/entitydetails/components/EntityNavigation.js

```diff
@@ -0,0 +1,66 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+
+import './EntityNavigation.css';
+
+import type { DbEntity } from 'modules/entitieslist';
+
+
+type Props = {|
+    +entity: DbEntity,
+|};
+
+
+/**
+ * Component showing entity navigation toolbar.
+ *
+ * Shows next/previous buttons.
+ */
+export default class EntityNavigation extends React.Component<Props> {
+    goToNextEntity = (entity: DbEntity) => {
+        console.log("foo");
```

Comment:
@adngdb For some reason I see the log messages while the Translate view is loading. Clicking on Previous/Next buttons OTOH doesn't trigger them. What am I doing wrong? :)

Reply:
This happens because you actually call these functions when you render the component. And when you click, it calls the return value of these functions, which is `undefined`, so it does nothing. You don't need to pass the entity here, as that variable is a prop that's always the same and always available in this component. 

To fix your problem, remove `entity` from the parameters of your `goTo` methods and get it from the props instead, and don't call the methods when you pass them in `onClick`.

Reply:
Btw, if you haven't seen it already, here's a function that might come in handy: https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/entitieslist/selectors.js#L52